Packs/CortexXDR/TestPlaybooks/Test_Playbook_-_Cortex_XDR_Malware_-_Incident_Enrichment.yml

```diff
@@ -0,0 +1,4565 @@
+            value:
+              complex:
+                root: Endpoint
+                accessor: MACAddress
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 6970,
+          "y": -380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "179":
+    id: "179"
+    taskid: 01613bc3-f6fa-4e82-8155-f3aca5aefcf8
+    type: condition
+    task:
+      id: 01613bc3-f6fa-4e82-8155-f3aca5aefcf8
+      version: -1
+      name: Verify Device Name
+      description: |
+        Verify that the ‘devicename’ incident field was filled out correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "186"
+      Verified:
+      - "136"
+    separatecontext: false
+    conditions:
+    - label: Verified
+      condition:
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: incident.devicename
+            iscontext: true
+          right:
+            value:
+              complex:
+                root: Endpoint
+                accessor: Hostname
+            iscontext: true
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 7370,
+          "y": -380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "180":
+    id: "180"
+    taskid: 3f7806ed-77f7-4aa6-8c3a-2800336f36b0
+    type: condition
+    task:
+      id: 3f7806ed-77f7-4aa6-8c3a-2800336f36b0
+      version: -1
+      name: Verify Device OS Name
+      description: |
+        Verify that the ‘deviceosname’ incident field was filled out correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "184"
+      Verified:
+      - "136"
+    separatecontext: false
+    conditions:
+    - label: Verified
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: incident.deviceosname
+            iscontext: true
+          right:
+            value:
+              complex:
+                root: Endpoint
+                accessor: OS
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 5760,
+          "y": -380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "182":
+    id: "182"
+    taskid: 3511a1b2-fd06-4375-87a4-af6be9ca684d
+    type: condition
+    task:
+      id: 3511a1b2-fd06-4375-87a4-af6be9ca684d
+      version: -1
+      name: Verify Device Status
```

Comment:
duplicated with task 207. 
Note that it is not the only one... also 183<->200
and 178<->204 and some more

Reply:
Removed the following duplicated tasks:
 1- 'Verify Device Status' - task number 182
2- 'Verify Isolated' - task number 183
3- 'Verify Device Name' - task number 179
4- 'Verify Device MAC Address' -task number  178
5- 'Verify Device OS Name' - task number 180
6- 'Verify Device ID' - task number 176
7 - 'Verify Device Local IP' - task number 177